src/main/java/net/sf/jabref/gui/actions/OpenBrowserAction.java

```diff
@@ -0,0 +1,49 @@
+package net.sf.jabref.gui.actions;
+
+import java.awt.event.ActionEvent;
+
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import javax.swing.Icon;
+
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+
+public class OpenBrowserAction extends AbstractAction {
```

Comment:
I think the UpdateDialog also has some actions which might be replaced by this one.